### PR TITLE
kraken: fix disruptions in departures for realtime

### PR DESCRIPTION
**Pull request by TeXitoi**
related to http://jira.canaltp.fr/projects/NAVITIAII/issues/NAVITIAII-2202

### CODE REVIEW

**Comment by TeXitoi** [comment]
Jenkins, test this please.

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 1]
```diff
@@ -234,6 +255,17 @@ def _check_train_delay_disruption(dis):
         eq_(get_arrivals(new_base), ['20120614T080222', '20120614T080436'])
         eq_(get_used_vj(new_base), [['vjA'], []])
 
+        # we have one delayed departure
+        departures = self.query_region("stop_points/stop_point:stopB/departures?_current_datetime=20120614T0800", display=True)
+        assert len(departures['disruptions']) == 2
+        assert len(departures['departures']) == 2
+        assert departures['departures'][0]['stop_date_time']['departure_date_time'] == '20120614T080225'
+        # only one more disruption in base_schedule
```
Comment (antoine-de) [inline source/jormungandr/tests/kirin_realtime_tests.py]: hum maybe it would be clearer to say that we have one more disruption not compared to the real_time, but compare to before the dirsruption no ?

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 2]
```diff
@@ -260,12 +292,6 @@ def _check_train_delay_disruption(dis):
         eq_(get_arrivals(new_base), ['20120614T080222', '20120614T080436'])
         eq_(get_used_vj(new_base), [['vjA'], []])
 
-        # we then try to send a delay on another train.
```
Comment (antoine-de) [inline source/jormungandr/tests/kirin_realtime_tests.py]: why did you remove this ?

File: source/kraken/realtime.cpp [hunk 3]
```diff
@@ -252,8 +254,12 @@ create_disruption(const std::string& id,
                         departure_time = arrival_time = st.arrival().time();
                     }
                 }
-                auto ptime_arrival = bpt::from_time_t(arrival_time) - start_first_day_of_impact;
-                auto ptime_departure = bpt::from_time_t(departure_time) - start_first_day_of_impact;
+                const auto arrival_pt = bpt::from_time_t(arrival_time);
```
Comment (antoine-de) [inline source/kraken/realtime.cpp:265]: can you add a comment explaining this mechanism ?

File: source/jormungandr/tests/kirin_realtime_tests.py (hunk 2 above)
Comment (TeXitoi) [reply source/jormungandr/tests/kirin_realtime_tests.py]: Because it was late ;-)

File: source/kraken/realtime.cpp (hunk 3 above)
Comment (TeXitoi) [reply source/kraken/realtime.cpp:265]: Comment added.

**Review by antoine-de — APPROVED** [review]
